src/server/game/Entities/Player/Player.cpp

```diff
@@ -22069,17 +22069,17 @@ bool Player::ActivateTaxiPathTo(std::vector<uint32> const& nodes, Creature* npc
     // taximaster case
     if (npc)
     {
+
         // not let cheating with start flight mounted
         if (IsMounted())
         {
-            GetSession()->SendActivateTaxiReply(ERR_TAXIPLAYERALREADYMOUNTED);
-            return false;
+            Dismount();
+            RemoveAurasByType(SPELL_AURA_MOUNTED);
```

Comment:
Why do you use both Dismount and RemoveAurasByType?

Reply:
Yes, maybe only  RemoveAurasByType(SPELL_AURA_MOUNTED); will work.

Reply:
Fixed